client/landing/subscriptions/components/site-list/styles.scss

```diff
@@ -74,6 +74,19 @@ $max-list-width: 1300px;
 					}
 				}
 			}
+
+			.p2-label {
+				display: inline-block;
+				text-align: center;
+				width: 36px;
+				height: 20px;
+				background: #2a6aee;
```

Comment:
I think there still isn't 100% agreement on what color should be used: p1682440802321329-slack-C02TCEHP3HA.

Reply:
We reached a consensus. Applying the correct one now.